components/DropdownSearch/index.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import placeholder from "../../public/icon-location.png";
+
+const NOMNATIM_BASE_URL = "https://nominatim.openstreetmap.org/search?";
```

Comment:
typo
```suggestion
const NOMINATIM_BASE_URL = "https://nominatim.openstreetmap.org/search?";
```

additionally, it would be great to have this constant in some common constants, as I suppose it can be used not just in one place